mitodl/ol-keycloak
# Adjustments to theme to match latest UX

### What are the relevant tickets?




Closes https://github.com/mitodl/hq/issues/5338

### Description (What does it do?)

This addresses a variety of style tweaks to align the UX to the latest UX mockups.

Note that I also had to do some hackery with javascript to get the recaptcha to size correctly. This won't exactly match the mockups because we don't have control over the recaptcha CSS itself, so the best we can do is scale it as best as possible.

### Screenshots (if appropriate):

![Screenshot 2024-09-06 at 15-05-56 MIT Learn](https://github.com/user-attachments/assets/b18d1282-3ab2-4db5-ba4b-15478058f60b)

![Screenshot 2024-09-06 at 14-58-50 MIT Learn](https://github.com/user-attachments/assets/f83a79c8-b04c-44c2-a823-5be1914559f0)


### How can this be tested?

- Go to the login page and verify it looks like the mockups
- Go to the registration page and verify it looks like the mockups and that the recaptcha scales over various browser widths
